airgun/views/host_new.py

```diff
@@ -69,10 +69,10 @@ class Card(View):
     title = Text('.//div[@class="pf-v5-c-card__title"]')
 
 
-class DropdownWithDescripton(Dropdown):
+class DropdownWithDescripton(PF5Dropdown):
```

Comment:
```suggestion
class DropdownWithDescription(PF5Dropdown):
```

Reply:
I will keep it separate as this needs to be cherrypicked to older versions.

Reply:
All right